src/show/pages/private/MyProjects/index.js

```diff
@@ -0,0 +1,132 @@
+import axios from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { Fragment, useState, useEffect } from 'react';
+import { FaPencilAlt, FaTrash } from 'react-icons/fa';
+
+import {
+  EDIT_PROJECT_PATH,
+  PROJECT_PATH
+} from '../../../../process/routes/paths';
+import { Button, ProgressBar, Modal } from '../../../components';
+
+const MyProjects = () => {
+  const [projectId, setProjectId] = useState(null);
+  const [myProjects, setMyProjects] = useState([]);
+  const [projectModal, setProjectModal] = useState(false);
```

Comment:
Where's the reference that you delete a project through this modal?
deleteProjectModal